osquery/events/tests/events_database_tests.cpp

```diff
@@ -153,7 +153,7 @@ TEST_F(EventsDatabaseTests, test_record_range) {
 
   for (size_t j = 0; j < 30; j++) {
     // 110 is 10 below an index (60.2).
-    sub->testAdd(110 + j);
+    sub->testAdd(110 + static_cast<int>(j));
```

Comment:
Do you mean for `testAdd`? Do we not use the `INTEGER` macro in `testAdd`? And also, should `testAdd(-130 + 140)` work?